DiamondLightSource/python-artemis
# Ensure proper use of class variables

Fixes #810

Link to dodal PR (if required): https://github.com/DiamondLightSource/dodal/pull/123

### To test:
1. Confirm class variables are removed and tests still pass
2. Confirm exceptions are listed in the issue
